index.html

```diff
@@ -114,6 +114,96 @@ <h3 class="skills">Skills</h3>
+            </ul>
+            <div class="button">
+               <a href="" class="main-button">See Project</a>
+            </div>
+         </div>
+      </div>
+      <div class="work1">
+         <div class="img1">
+            <img src="images/img.png" class="image" alt="software-portfolio">
+         </div>
+         <div class="info1">
+            <h3 class="sub1">Multi-Post Stories</h3>
+            <div class="circle4"></div>
+            <p class="desc1">
+               A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <ul class="main-works">
+               <li class="works">css</li>
+               <li class="works">html</li>
+               <li class="works">bootstrap</li>
+               <li class="works none">Ruby</li>
+            </ul>
+            <div class="button">
+               <a href="" class="main-button">See Project</a>
+            </div>
+         </div>
+      </div>
+      <div class="work1">
+         <div class="img1">
+            <img src="images/img.png" class="image" alt="software-portfolio">
+         </div>
+         <div class="info1">
+            <h3 class="sub1">Multi-Post Stories</h3>
+            <div class="circle5"></div>
+            <p class="desc1">
+               A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <ul class="main-works">
+               <li class="works">css</li>
+               <li class="works">html</li>
+               <li class="works">bootstrap</li>
+               <li class="works none">Ruby</li>
+            </ul>
+            <div class="button">
+               <a href="" class="main-button">See Project</a>
+            </div>
+         </div>
+      </div>
+      <div class="work1">
+         <div class="img1">
+            <img src="images/img.png" class="image" alt="software-portfolio">
+         </div>
+         <div class="info1">
+            <h3 class="sub1">Multi-Post Stories</h3>
+            <div class="circle6"></div>
+            <p class="desc1">
+               A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <ul class="main-works">
+               <li class="works">css</li>
+               <li class="works">html</li>
+               <li class="works">bootstrap</li>
+               <li class="works none">Ruby</li>
+            </ul>
+            <div class="button">
+               <a href="" class="main-button">See Project</a>
+            </div>
+         </div>
+      </div>
+   </section>
+   

```

Comment:
- [ ] For this milestone you need to implement the `About myself` section, which is missing right now, so please make sure to add this section as well.